Gemfile

```diff
@@ -31,6 +31,7 @@ gem 'rails_admin'
 gem 'rdf'
 gem 'rdf-rdfxml'
 gem 'rdf-vocab'
+gem 'recaptcha', require: 'recaptcha/rails'
```

Comment:
This seems to break the Devise sign-in form at /users/sign_in, for me at least. After submitting correct email and password, I get the error:
```
No route matches [PATCH] "/users/sign_in"
```

Reply:
I can't reproduce the error, the Issue you're having might be unrelated to these changes